ExternalTasks/NodeJs/main.js

```diff
@@ -1,70 +1,68 @@
 const {HttpClient} = require('@essential-projects/http');
+
 const {
   ExternalTaskApiClientService,
   ExternalTaskApiExternalAccessor,
   ExternalTaskWorker,
 } = require('@process-engine/external_task_api_client');
+const {ExternalTaskFinished} = require('@process-engine/external_task_api_contracts');
 
-const {
-  ExternalTaskFinished,
-} = require('@process-engine/external_task_api_contracts');
-
-const identity = {
-    token: 'ZHVtbXlfdG9rZW4=',
+const sampleIdentity = {
```

Comment:
Sollten wir hier evtl. einen Kommentar hinzufügen, wie bspw.

```suggestion
const sampleIdentity = {
  // The following is a "dummy token" that always works with the Studio's internal ProcessEngine
```

um dem geneigten Leser die Bedeutung dieses Sonderfalls klar zu machen?

Reply:
Klingt gut.